api/proto/info/info.proto

```diff
@@ -201,6 +201,12 @@ enum ZSwState {
   // state. If an App Instance is in ERROR state, it means there is NO DOMAIN
   // currently running.
   ERROR                 = 21;   // Error State
+
+  // It would have been ideal to leave a gap before the ERROR state.
+  // Since we cannot change the existing states now, introduce the
+  // Z_SW_STATE_START_DELAYED state after ERROR.
+  // Had to prefix with Z_SW_STATE_ to satisfy yetus checks.
```

Comment:
We tend to ignore the yetus complaints when adding an enum value to an existing enum to instead have them be internally consistent, so START_DELAYED makes more sense.

As we implement this does it make sense to use the Progress to indicate the percentage of time which is left?

Reply:
Will change the name to START_DELAYED